bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/LargeDataActivity.kt

```diff
@@ -50,13 +50,13 @@ class LargeDataActivity : BridgeBaseActivity() {
 
         fun getArguments(
             activity: LargeDataActivity
-        ): LargeDataActivityArguments = activity
+        ): LargeDataActivityArguments? = activity
```

Comment:
So I'm guessing this is because when navigating from a deep link we won't have this argument? In that case what we should do is create it so that we can return a non-null value here.

Reply:
yeah.. that's for test, I could split to anther activity